cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -461,3 +462,44 @@ def test_reports_online_vms(appliance, setup_provider, provider, request, vm):
 
     view = navigate_to(saved_report, "Details")
     assert vm not in [row.vm_name.text for row in view.table.rows()]
+
+
+@pytest.mark.tier(0)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.uncollectif(
+    lambda case_sensitive: not case_sensitive
+    and BZ("1741588", forced_streams=["5.11"]).blocks
+)
+@pytest.mark.parametrize(
+    "case_sensitive", [True, False], ids=["case-sensitive", "case-insensitive"]
+)
+@pytest.mark.meta(automates=[1678150, 1741588])
+def test_reports_filter_content(
+    appliance, case_sensitive, set_and_get_tenant_quota, tenant_report
+):
+    """
+    Bugzilla:
+        1678150
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/3h
+        startsin: 5.11
+        setup:
+            1. Go to Cloud Intel -> Reports -> All Reports
+            2. Select a report and queue it, make sure it's not empty.
+        testSteps:
+            1. Add a filter.
+            2. Traverse through all the rows and check if the content is filtered.
+        expectedResults:
+            1.
+            2. Content must be filtered.
+    """
+    search_term = "in GB" if case_sensitive else "in gb"
```

Comment:
I was not sure which report I was going to use for the test, which is why I kept the parameters generic. I wouldn't have had to change the parameters in case I wanted to use a different report.